src/screens/QuestionScreen.tsx

```diff
@@ -78,7 +78,7 @@ function getElement(questionOption: QuestionOption) {
     case QuestionOptionType.OPEN:
       return (
         <OpenTextArea
-          defaultValue={questionOption.answer?.values?.[0] ?? ''}
```

Comment:
acc label "Antwoord veld" of "Voer antwoord in"

Reply:
Verbetering: "Open antwoord veld"